packages/utils/src/utils/formatters.ts

```diff
@@ -36,16 +37,16 @@ export const safeFormatUnits = (value: BigNumberish, decimals: number | string =
  */
 export const formatVisualAmount = (
   value: BigNumberish,
-  decimals: number | string = GWEI,
+  decimals?: number | string | null,
   precision?: number,
 ): string => {
-  const amount = safeFormatUnits(value, decimals)
+  const amount = safeFormatUnits(value, decimals ?? GWEI)
   return precision !== undefined ? formatAmountPrecise(amount, precision) : formatAmount(amount)
 }
 
-export const safeParseUnits = (value: string, decimals: number | string = GWEI): bigint | undefined => {
+export const safeParseUnits = (value: string, decimals?: number | string | null): bigint | undefined => {
   try {
-    return parseUnits(value, decimals)
+    return parseUnits(value, decimals ?? GWEI)
```

Comment:
Nit: It's subjective, but default values are clearer at the top of the function (whether as a fallback or reassignment). I'll leave this up to your discretion though.

Reply:
Same as with validation.ts.